frontend/src/features/videos/VideoCard.tsx

```diff
@@ -218,9 +220,18 @@ function VideoCard({
               {video.publication_date}
             </span>
           )}
-          {video.uploader && (
-            <span className={classes.channel}>{video.uploader}</span>
-          )}
+          <Tooltip title="Filter by channel" placement="bottom">
```

Comment:
For consistency with the `UploaderFilter`, "Filter by uploader" could be a more appropriate title.